src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -92,7 +92,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                              _CGenJit=$(_JitPath);
                              _CGenInputAssembly=%(_AssembliestoCrossGen.Fullpath);
                              _CGenOutputAssembly=$([System.IO.Path]::Combine($(_RuntimeOptimizedDir),%(RecursiveDir)%(Filename)%(Extension)));
-                             _CGenPlatformAssembliesPath=$(_NetCoreRefDir)$(PathSeparator)$(_RuntimeRefDir);" />
+                             _CGenPlatformAssembliesPath=$(_RuntimeRefDir)$(PathSeparator)$(_NetCoreRefDir);" />
```

Comment:
Giving the app assemblies the priority again as system assemblies are filtered out in the common case